core/src/main/java/org/apache/accumulo/core/client/mapreduce/lib/impl/InputConfigurator.java

```diff
@@ -517,6 +517,74 @@ public static Boolean isOfflineScan(Class<?> implementingClass, Configuration co
   }
 
   /**
+   * Controls the use of the {@link BatchScanner} in this job.
+   * Using this feature will group ranges by their source tablet per InputSplit and use BatchScanner to read them.
```

Comment:
Could mention what options this will not work with in javadoc.  For example setting this will not work w/ offline scan and local iters.

Could also mention the benefit of doing this.  It seems the main benefit is one RPC to send multiple ranges to a tserver and then read the results, versus an RPC per range.